### DISCUSSION TITLE
Works on Ubuntu on Windows. <3

### DISCUSSION THREAD

**Discussion by vanhecke** [postmodern/ruby-install#495] [Show and tell]
Since Ubuntu still manages to ship Ruby 1.9.3 I instantly came back to this script.
Just worked as expected, pretty cool.

( I only tested regular ruby )

**Comment by FranklinYu** [comment]
On Windows?

**Comment by vanhecke** [comment]
@franklinyu "Bash on Ubuntu on Windows"*

**Comment by FranklinYu** [comment]
@Vanhecke Wow, good to hear that. It was a nightmare to install Ruby on Windows.
